workspaces/rbac/plugins/rbac-backend/src/database/role-metadata.ts

```diff
@@ -65,6 +68,25 @@ export class DataBaseRoleMetadataStorage implements RoleMetadataStorage {
     });
   }
 
+  async filterForOwnerRoleMetadata(
+    filter?: RBACFilters,
+  ): Promise<RoleMetadataDao[]> {
+    let filteredRoleMeta: RoleMetadataDao[];
+
+    const roleMetadata: RoleMetadataDao[] =
+      await this.knex.table(ROLE_METADATA_TABLE);
+
+    if (filter) {
+      filteredRoleMeta = roleMetadata.filter(role => {
+        return matches(role as RoleMetadata, filter);
+      });
+    } else {
+      filteredRoleMeta = roleMetadata;
```

Comment:
I think you can use direct return here:

return roleMetadata;

Reply:
done